api/src/main/java/org/openmrs/module/ugandaemr/UgandaEMRConstants.java

```diff
@@ -174,14 +174,14 @@ public class UgandaEMRConstants {
     public static final String HIV_PROGRAM_UUID = "18c6d4aa-0a36-11e7-8dbb-507b9dc4c741";
     public static final String HIV_DISCONTINUATION_FORM_NAME = "HIV Discontinuation";
     public static final String HIV_DISCONTINUATION_FORM_UUID ="1c7baac0-87cf-4d12-a938-81ea0d6f6448";
-    public static final String HIV_ENROLLMENT_FORM_NAME ="HIV Enrollment";
-    public static final String HIV_ENROLLMENT_FORM_UUID ="4d933e88-27fb-320e-9323-14c5d35ea1ee";
+    public static final String HIV_ENROLLMENT_FORM_NAME ="HMIS ACP HIV 003: HIV CARE/ART CARD - Summary Form";
+    public static final String HIV_ENROLLMENT_FORM_UUID ="2efbb08f-e6df-3791-bbd7-8de42eeb9cd7";
```

Comment:
The actual form uuif form the HIV Enrollment form is 52653a60-8300-4c13-be4d-4b746da06fee